site/source/docs/web3js.md

```diff
@@ -1,5 +1,4 @@
 title: Web3.js Documentation
 ---
 
-The documentation for Web3.js can be found [here](http://web3js.readthedocs.io/en/1.0/)
-
+The documentation for Web3.js can be found [here](https://web3js.readthedocs.io/en/v1.2.0/)
```

Comment:
This document is no longer reachable through our site and can actually be thrown away. But we can do that in a separate PR.